tests/testsuite.src/listings.at

```diff
@@ -3566,6 +3566,73 @@ AT_CHECK([diff prog20.lst prog.lis], [0], [], [])
 AT_CLEANUP
 
 
+AT_SETUP([MFCOMMENT])
+AT_KEYWORDS([listing])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       PROCEDURE        DIVISION.
+      *    DISPLAY 'COMMENTASTERISK'
+      /    DISPLAY 'COMMENTSLASH'
+*          DISPLAY 'MFCOMMENTASTERISK'
+/          DISPLAY 'MFCOMMENTSLASH'
```

Comment:
There is no MFCOMMENTSLASH thing, neither in MF's old or new docs; and it also is only seen as sequence-number (fixed/variable) or as normal print after line-break indicator in xopen_free.
Please drop that from _all_ tests (I've added it back then because I've seen it in the code - I should have checked the docs and then asked for people with MF access... - instead; but that's now done).